routes/admin/portals/control/registration.js

```diff
@@ -0,0 +1,14 @@
+let express = require("express");
```

Comment:
This is a public facing endpoint. This should not be present in `routes/admin/portals/control`

Reply:
Also, why was this change needed? We are not using this router anywhere.

Reply:
As discussed offline, please remove this file in a follow-up PR.